src/clarity-angular/datagrid/datagrid.html

```diff
@@ -36,7 +36,7 @@
 
     <!-- Custom placeholder overrides the default empty one -->
     <ng-content select="clr-dg-placeholder"></ng-content>
-    <clr-dg-placeholder *ngIf="!placeholder"></clr-dg-placeholder>
+    <clr-dg-placeholder *ngIf="!placeholder && items.displayed.length == 0"></clr-dg-placeholder>
```

Comment:
But, with the flex layout and the scrolling implementation, won't the grid always be as high, as defined by its container?
For me, setting the datagrid height in number of rows was a problem, because row height can not be predicted due to word wrapping, etc.
Before you introduced the internal datagrid scrolling, I had to adopt the rather questionable approach of having page size that's higher than the viewport, and scrolling outside of the datagrid. That's how gmail is implemented, as a reference.
So, my point is, isn't the correct use case now to have a fixed height on the datagrid, but in px, %, vh, whatever, and scrolling inside the datagrid itself?